nncf/openvino/engine.py

```diff
@@ -15,70 +15,77 @@
 from typing import Callable
 from typing import Iterable
 from typing import Any
-from time import time
+from typing import List
 
-from openvino.runtime import AsyncInferQueue
 from openvino.tools import pot
 
 from nncf.api.compression import TModel
 from nncf.data import Dataset
 
 
-logger = pot.utils.logger.get_logger(__name__)
-
-
 # TODO(andrey-churkin): This class should be removed after refactoring of the OVEngine class.
 # We will be able to do that when we will have moved the POT code in the NNCF.
-class DummyMetric(pot.Metric):
+class DummyMetric:
```

Comment:
Why do we have an implementation of `Metric` and `DatasetWrapper` in the file named `engine.py`?

Reply:
Good question! I guess that these entities should not be used outside this file. They should be removed in the future (it is used only to integrate POT to NNCF). So I don't actually see the reasons to move it to another module.